python/sdk/tests/test_iam.py

```diff
@@ -15,183 +13,65 @@ class TestIAM:
-            mode=Mode.MANUAL.value,
-            controls=[],
-            users=[
-                dict(
-                    handle=email,
-                    permission=Permission.ADMIN.value,
-                    name="Bob",
-                    subscriptions=[],
-                    oidc=False,
-                    terms={},
-                )
-            ],
-            probe_sleep="600s",
-            oidc=dict(),
-            features=dict(
-                threat_intel=False,
-                detect=False,
-                detections=True,
-                policy_evaluator=True,
-            ),
-        )
-        pytest.expected_account["users"][0]["expires"] = created["users"][0]["expires"]
-        pytest.expected_account["queue"] = created["queue"]
-        if created["queue"]:
-            assert "autopilot" == created["queue"][0]["tag"]
-            assert RunCode.SMART.value == created["queue"][0]["run_code"]
-
-    def test_account_subscribe(self, unwrap):
-        res = unwrap(self.iam.subscribe)(self.iam, event=self.subscription_event)
-
-        for user in pytest.expected_account["users"]:
-            if user["handle"] == pytest.expected_account["whoami"]:
-                user["subscriptions"].append(self.subscription_event.value)
-                self.expected_subs = user["subscriptions"]
-                break
-
-        expected = dict(
-            email=pytest.expected_account["whoami"], subscriptions=self.expected_subs
-        )
-        diffs = check_dict_items(expected, res)
-        assert not diffs, json.dumps(diffs, indent=2)
+        self.service_user = "registration"
 
     def test_get_account(self, unwrap):
         res = unwrap(self.iam.get_account)(self.iam)
         diffs = check_dict_items(pytest.expected_account, res)
         assert not diffs, json.dumps(diffs, indent=2)
 
-    def test_create_user(self, unwrap):
-        ex = datetime.now(timezone.utc) + timedelta(days=1)
-        res = unwrap(self.iam.create_user)(
-            self.iam,
-            email=self.second_user,
-            permission=Permission.SERVICE,
-            name="Rob",
-            expires=ex,
+    def test_list_accounts(self, unwrap):
+        res = unwrap(self.iam.list_accounts)(self.iam)
+        assert pytest.expected_account["account_id"] in [a["account_id"] for a in res]
+
+    def test_service_user(self, unwrap):
+        service_user = unwrap(self.iam.create_service_user)(
+            self.iam, handle=self.service_user
         )
-        assert self.second_user == res["handle"]
-        assert check_if_string_is_uuid(res["token"])
+        assert self.service_user == service_user["handle"]
+        assert check_if_string_is_uuid(service_user["token"])
 
         res = unwrap(self.iam.get_account)(self.iam)
 
-        pytest.expected_account["users"].append(
+        pytest.expected_account["token_users"].append(
             dict(
-                handle=self.second_user,
+                handle=self.service_user,
```

Comment:
i shifted the "handle" to the `name` field and am now generating random uuids for the handle